include/uart.h

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
Minor, but why is this put on top of existing copyright, instead of on bottom?

Reply:
Just following approach in other API .h files

Reply:
Weird, never paid attention. Definitely not how Linux has it, e.g. https://elixir.bootlin.com/linux/latest/source/ipc/sem.c (though not too consistent there too, e.g. 2010 and 2016 swapped in this random example.)